pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -108,7 +112,6 @@ buildDotnetModule rec {
   };
```

Comment:
How should the tests reference the package's exe without explicitly referencing `pkgs.nexusmods-app`?

Presumably this may use the wrong derivation when we're building an overridden package?

Reply:
Seems most other packages just set `nativeBuildInputs = [ nexusmods-app ]` in the test derivations.

Maybe the `-unfree` override in all-packages.nix should also be overriding the `nexusmods-app` argument passed in?

Reply:
I'm going to look at [these examples](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20finalAttrs.finalPackage&type=code) using the `finalAttrs` pattern tomorrow if I get chance

Reply:
Sorry, I responded to the other thread pretty much saying what you said here.